.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/zeetec/project/hamlab/commitlint.config.js":"1","/Users/zeetec/project/hamlab/pages/index.tsx":"2","/Users/zeetec/project/hamlab/postcss.config.js":"3","/Users/zeetec/project/hamlab/tailwind.config.js":"4"},{"size":66,"mtime":1671747458432,"results":"5","hashOfConfig":"6"},{"size":537,"mtime":1671751788617,"results":"7","hashOfConfig":"6"},{"size":82,"mtime":1671751788618,"results":"8","hashOfConfig":"6"},{"size":239,"mtime":1671751788618,"results":"9","hashOfConfig":"6"},{"filePath":"10","messages":"11","suppressedMessages":"12","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1gnlqwg",{"filePath":"13","messages":"14","suppressedMessages":"15","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","suppressedMessages":"18","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"19","messages":"20","suppressedMessages":"21","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/zeetec/project/hamlab/commitlint.config.js",[],[],"/Users/zeetec/project/hamlab/pages/index.tsx",[],[],"/Users/zeetec/project/hamlab/postcss.config.js",[],[],"/Users/zeetec/project/hamlab/tailwind.config.js",[],[]]
```

Comment:
I think this file should be listed in `.gitigonore` since we just want to see the cache locally.

Reply:
For adding this file in gitignore, I will do in different commit